README.md

```diff
@@ -23,36 +23,45 @@ Environment variables(s):
   ``` javascript
      const SlackHooks = require('notification-hooks').Slack
   
-     const text = 'Hello world!'      // supports both string & object
-     const channel = '#media-upload'
-     const username = '@albert @ryan'       
-     const icon_emoji = ':lion_face:' // defaults to - ':fire:'
+     const options = {
+        text: 'Hello world!', // supports both string & object
+        channel: '#media-upload',
+        username: '@albert @ryan',
+        icon_emoji: ':lion_face:', // defaults to - ':fire:'
+        code: true // supports code and non code styles   
```

Comment:
'code' is not obvious property name, you can change to 'codeSnippet' as in slack